OpenRA.Game/Translation.cs

```diff
@@ -117,7 +117,11 @@ public bool TryGetString(string key, out string value, IDictionary<string, objec
 
 		public bool HasMessage(string key)
 		{
-			return bundle.HasMessage(key);
+			var attributes = key.LastIndexOf('.');
+			if (attributes > 0)
+				return bundle.HasMessage(key.Remove(attributes));
+			else
+				return bundle.HasMessage(key);
```

Comment:
This won't catch missing attributes because it checks the parent key. Testcase: remove an attribute from a key (e.g. a `fog-of-war.label`) and run the lint. In my implementation I was using `bundle.TryGetAttrMsg(key)` and it worked however it doesn't seem to work here anymore.

Reply:
For me this catches missing attributes
```C#
return bundle.TryGetAttrMsg(key, null, out _, out _);
```

Reply:
This was fixed upstream. https://github.com/Ygg01/Linguini/pull/26 :star_struck: